README.md

```diff
@@ -432,7 +432,7 @@ Auth0
                     DispatchQueue.main.async {
                         Auth0
                             .webAuth()
-                            .parameters(["realm": realm,
+                            .parameters(["connection": realm,
```

Comment:
The thing is, there is already a builder method for "connection". Let me test the snippet with that method and get back to you.

Reply:
I tested the snippet with the `connection` method, and updated the snippet in https://github.com/auth0/Auth0.swift/pull/410/commits/412b4c0be58cd22dd2d2ccf4a14c331ac1e7db8b.